app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class UserMailer < ApplicationMailer
+  def email_confirmation(user, email)
+    @user = user
+    mail to: email, subject: 'Email confirmation'
```

Comment:
Consider something like "GAME_NAME: Email confirmation"

Paging @ChristopherJMiller do we have a definitive name for this project other than "Wizard Connect 3"?

Reply:
Changed to 'Wizard Connect 3: Email Confirmation' for now.

Reply:
No definitive name yet, Wizard Connect 3 is fine for now